packages/browser/src/transports/fetch.ts

```diff
@@ -17,6 +17,11 @@ export function makeFetchTransport(
       method: 'POST',
       referrerPolicy: 'origin',
       headers: options.headers,
+      // Outgoing requests are usually cancelled when navigating to a different page, causing a "TypeError: Failed to
+      // fetch" error and sending a "network_error" client-outcome. In chrome the request status shows "(cancelled)".
+      // The `keepalive` flag keeps outgoing requests alive, even when switching pages. We want this since we're
+      // frequently sending events (i.e.navigation transactions) right before the user is switching pages.
+      keepalive: true,
```

Comment:
Can we make this change in another PR if we're gonna go ahead with it - ref: ref: https://github.com/getsentry/sentry-javascript/issues/2547